requirements/AOCS-GNC_AURORA_0001/taste_project/InterfaceView.aadl

```diff
@@ -0,0 +1,706 @@
+---------------------------------------------------
```

Comment:
This file is automatically generated - please remove

Reply:
Done